frontend/src/common/Placeholder.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+    Button,
+    MenuList,
+    MenuListItem,
+    Separator,
+    Window,
+    WindowContent,
+    WindowHeader,
+} from 'react95';
+import {Link} from 'react-router-dom';
+import Draggable from 'react-draggable';
+
+type PlaceholderProps = {
+    title: string;
+};
+const Placeholder = ({title}: PlaceholderProps) => {
+    return (
+        <Draggable>
+            <Window>
+                <WindowHeader>Placeholder dla strony {title}</WindowHeader>
+                <WindowContent>
+                    <MenuList inline>
+                        {['/', '/login', '/events', '/payments'].map((path, i) => (
```

Comment:
we already have list of routes in `App.tsx`, try to use it as single source of truth